src/shapes/text.class.ts

```diff
@@ -1609,10 +1609,8 @@ export class Text<
         ? style.fontFamily
         : `"${style.fontFamily}"`;
     return [
-      // node-canvas needs "weight style", while browsers need "style weight"
-      // verify if this can be fixed in JSDOM
-      fabric.isLikelyNode ? style.fontWeight : style.fontStyle,
-      fabric.isLikelyNode ? style.fontStyle : style.fontWeight,
+      style.fontStyle,
+      style.fontWeight,
```

Comment:
?

Reply:
this is not true anymore this was fixed long time ago in node-canvas

Reply:
https://github.com/Automattic/node-canvas/issues/1620